docs/docs-content/clusters/data-center/nutanix/nutanix.md

```diff
@@ -8,36 +8,48 @@ tags: ["data center", "nutanix"]
+  :::
 
-  When the PCG status displays Running state, the Kubernetes cluster is integrated with Palette.
+
+2. Create a Nutanix self-hosted PCG by installing Palette agents in your existing Kubernetes cluster using kubectl commands provided in the Palette UI. For guidance, review [Install Private Cloud Gateway](install-pcg.md).
+
+  When the PCG status displays the state **Running**, the workload cluster is integrated with Palette.
 
   :::info
 
-  All clouds require the deployment of a PCG, which enables Palette to securely monitor clusters in the cloud. The PCG relies on the self-hosted Kubernetes cluster to remain operational.
+  Nutanix cloud requires the deployment of a PCG, which enables Palette to monitor clusters in the infrastructure provider environment. The PCG instance is installed on an existing Kubernetes cluster that must remain operational.
 
   :::
 
-3. Next, add the cloud account to Palette. The required fields to add the cloud are pre-defined by the system administrator during cloud registration in step 1. The PCG name provided at the time of PCG creation is auto-filled, and you provide the account name and account details.
+3. Next, add the Nutanix cloud account to Palette. Use the **drop-down Menu** to select the PCG name you provided when you configured it in the previous step. You must fill out the account name and account details.
 
 4. Create a cluster profile by selecting Nutanix as the cloud type. 
 
-  Palette displays out-of-the-box packs for the network and storage profile layers. Additionally, Palette provides a Nutanix-specific storage pack. If you have custom packs, you can add them to Palette by adding your registry. To learn how to add a pack registry, review [Add a Custom Registry](https://docs.spectrocloud.com/registries-and-packs/adding-a-custom-registry/#configure-a-custom-pack-registry-in-palette).
-
-5. Deploy a cluster by specifying Nutanix, listed under **Tech Preview**, as the cluster type. Select the cloud account you added and make any needed changes to the profile layers by using the YAML editor. At the **Cluster Config** step, adjust YAML configuration files to customize the cluster for your environment. You provide your own Operating System (OS) image.
+  When creating a Nutanix profile, you do not have to specify anything for the OS or Kubernetes layers. Palette provides out-of-the-box packs for the network and storage profile layers, including the **Nutanix CSI** storage pack. The **Nutanix CSI** pack is available when the cloud is registered using the name `nutanix`. If you have custom packs, you can add them to Palette by adding your registry. To learn how to add a pack registry, review [Add a Custom Registry](https://docs.spectrocloud.com/registries-and-packs/adding-a-custom-registry/#configure-a-custom-pack-registry-in-palette).
```

Comment:
This URL is incorect, use relative path instead of URL paths.

Reply:
:white_check_mark: